src/client/js/components/Admin/SlackIntegration/CustomBotWithoutProxySettings.jsx

```diff
@@ -21,6 +21,7 @@ const CustomBotWithoutProxySettings = (props) => {
   // get site name from this GROWI
   // eslint-disable-next-line no-unused-vars
   const [siteName, setSiteName] = useState('');
+  const [isBoltSetup, setIsBoltSetup] = useState(null);
```

Comment:
7回表示されるのは、まあ正しい挙動な感じがしますね

```suggestion
// eslint-disable-next-line no-unused-vars
  const [isBoltSetup, setIsBoltSetup] = useState(null);
```

log を出さなくでも、上記の対応で良いのでは？

Reply:
ESLintを無効にする方法に関して、勉強不足でした。確かにlogに出力する必要はないですね。